### ISSUE TITLE
I have 2 nearly identical centos servers. One mounted like a champ, the second is "unable to access MOUNTPOINT"

### ISSUE THREAD

**Issue by excenter** [s3fs-fuse/s3fs-fuse#782]
### Additional Information
On server 1, I followed the README, mounted the bucket, had a high five. 
On server 2, I replicated my commands from the first server, and it complains that 

> s3fs: unable to access MOUNTPOINT /home/centos/docker-volume: No such file or directory

despite there flagrantly being that directory with permissions ranging from 600 to 777

#### Version of s3fs being used (s3fs --version)
V1.83(commit:3e4002d) 
(on both)

#### Version of fuse being used (pkg-config --modversion fuse)
2.9.2
(on both) 

#### System information (uname -r)
3.10.0-693.21.1.el7.x86_64
(on both)

#### Distro (cat /etc/issue)
\S
Kernel \r on an \m
(on both) 

#### s3fs command line used (if applicable)
```
s3fs $BUCKET /home/centos/docker-volume -o passwd_file=/home/centos/.passwd-s3fs
```
#### s3fs syslog messages (grep s3fs /var/log/syslog, or s3fs outputs)  
_if you execute s3fs with dbglevel, curldbg option, you can get detail debug messages_
```
s3fs $BUCKET /home/centos/docker-volume -o passwd_file=/home/centos/.passwd-s3fs -o dbglevel=info -f -o curldbg

[CRT] s3fs.cpp:set_s3fs_log_level(272): change debug level from [CRT] to [INF]
s3fs: unable to access MOUNTPOINT /home/centos/docker-volume: No such file or directory
```

**Comment by excenter** [comment]
after deleting and remaking the docker_volume folder a number of times, it just worked.... 
Gift horses and all, assuming I misspelled something.

**Comment by gaul** [comment]
@excenter could you close this issue if you have resolved your symptoms?

**Comment by excenter** [comment]
closes #782
